geth/geth-genesis.json

```diff
@@ -13,6 +13,7 @@
     "muirGlacierBlock": 0,
     "berlinBlock": 0,
     "londonBlock": 0,
+    "shanghaiBlock": 8,  
```

Comment:
conflict here. It should be set to the latest `shardingForkBlock` value on master.

Reply:
thanks, fixed